posthog/hogql/functions/sparkline.py

```diff
@@ -4,9 +4,9 @@
 def sparkline(node: ast.Expr, args: list[ast.Expr]) -> ast.Expr:
     return ast.Tuple(
         exprs=[
-            ast.Constant(value="__hogql_chart_type"),
-            ast.Constant(value="sparkline"),
```

Comment:
Should we be getting rid of this existing method? I use it in error tracking (easily updated) but how can we know if there are other queries / insights relying on it

Reply:
we're not getting rid of it, it's just inlining the tag

Reply:
Ah yes, I re-read the description too